src/benchmarks/micro/Categories.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Data;
```

Comment:
why?

Reply:
I saw that as well. I have no idea how that got there.